internal/cli/atlas/atlas.go

```diff
@@ -49,11 +53,14 @@ const (
 )
 
 func Builder() *cobra.Command {
+	deprecatedMessage := newDeprecatedMessage()
 	opts := &cli.RefresherOpts{}
 	cmd := &cobra.Command{
 		Use:   Use,
 		Short: "MongoDB Atlas operations.",
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			_, _ = fmt.Fprintf(os.Stderr, `Command "%s" is deprecated, Atlas commands for MongoCLI are now deprecated but they will keep receiving support for 12 months (until April 30, 2023).
```

Comment:
[nit] "support" is to broad of a term, we should also avoid putting dates here as those dates may change, we should link to a docs page to explain what this deprecation means and with the timelines so we can update the page and older versions of the CLI don't show incorrect dates if they change

Reply:
FYI @dianchenghu as you proposed this message

Reply:
The copy we are suggesting was agreed upon by the docs and the marketing team.

For Mac users:
There’s a new, dedicated Atlas CLI available for Atlas users.  Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run brew install mongodb-atlas or visit [url]. Atlas commands for MongoCLI are now deprecated, but they will keep receiving support for 12 months (until April 30, 2023).
Non-Mac users:
There’s a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features: [url]. Atlas commands for MongoCLI are now deprecated, but they will keep receiving support for 12 months (until April 30, 2023).


Ideally, we could do the version above, but I was told that we have to start the string with "Command "atlas" is deprecated, " So updated version above. @boooczek